.gitignore

```diff
@@ -10,4 +10,4 @@ src/guides/bioinformatics/translate_ref.md
 src/guides/bioinformatics/vcf_input.md
 src/guides/communicate/create-pdf.md
 src/guides/communicate/narratives-intro.md
-src/tutorials/narratives-how-to-write.md
+src/tutorials/narratives-how-to-write-v1.md
```

Comment:
Is this needed? I didn't see `-v1.md` committed…

Reply:
Good catch - will remove.

For a while there I kept the previous ("v1") tutorial around, but in the end I felt it was more bad than good so removed it (almost) entirely

Reply:
Removed in https://github.com/nextstrain/docs.nextstrain.org/compare/03a1276b5dfebbb8d81a3447d64cd31239b9f4e4..477e27d73e3cde37e40c662e0b3aea0a9c4cb987